### PR TITLE
added binding appsody projects

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

In response to issue 292, `support binding existing project to appsody` from eclipse/codewind: https://github.com/eclipse/codewind/issues/292

I added a section, `Binding Appsody projects.`

### CODE REVIEW

**Comment by micgibso** [comment]
@makandre Please review

File: docs/_documentations/mdt-eclipse-importedprojects.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Codewind is designed to develop cloud native microservices. Therefore, each proj
 * [Swift projects](#swift-projects)
 * [Generic Docker projects](#generic-docker-projects)
 
+### Binding Appsody projects
```
Comment (makandre) [inline docs/_documentations/mdt-eclipse-importedprojects.md]: We should add an entry to the bullet list above

File: docs/_documentations/mdt-eclipse-importedprojects.md [hunk 2]
```diff
@@ -24,6 +24,11 @@ Codewind is designed to develop cloud native microservices. Therefore, each proj
...
+
+If you have existing Appsody project, or a project that you wish to add to Codewind as an Appsody project, select the corresponding Appsody project type and, if applicable, the Appsody stack when adding the project.
```
Comment (makandre) [inline docs/_documentations/mdt-eclipse-importedprojects.md]: If you have **an?** existing Appsody project...

**Comment by makandre** [comment]
Commented on the eclipse file, same comments also apply to vscode

File: docs/_documentations/mdt-eclipse-importedprojects.md (hunk 1 above)
Comment (j-c-berger) [reply docs/_documentations/mdt-eclipse-importedprojects.md]: Got it. Yes, makes total sense. I'll add `Appsody project` as a bullet entry. Thanks!

File: docs/_documentations/mdt-eclipse-importedprojects.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ Codewind is designed to develop cloud native microservices. Therefore, each proj
 * [Node.js projects](#nodejs-projects)
 * [Swift projects](#swift-projects)
 * [Generic Docker projects](#generic-docker-projects)
+* [Binding Appsody projects](#binding-appsody-projects)
```
Comment (makandre) [inline docs/_documentations/mdt-eclipse-importedprojects.md]: I think it's fine to leave out the word "Binding" here

Comment (j-c-berger) [reply docs/_documentations/mdt-eclipse-importedprojects.md]: Good call. I've made the change. Thanks.

**Pull request merged**